core/src/components/button/button.ios.scss

```diff
@@ -131,14 +126,53 @@
   --padding-bottom: #{$button-ios-round-padding-bottom};
 }
 
-
 // iOS Strong Button
 // --------------------------------------------------
 
 :host(.button-strong) {
   font-weight: #{$button-ios-strong-font-weight};
 }
 
+// iOS Icon Only Button
+// --------------------------------------------------
+
+:host(.button-has-icon-only) {
+  --padding-top: 0;
+  --padding-bottom: var(--padding-top);
+  --padding-end: var(--padding-top);
+  --padding-start: var(--padding-end);
+  // TODO(FW-6053): replace with the min-height variable.
+  min-width: 2.125em;
```

Comment:
Why are the min width/heights relative units?

Reply:
Discussed internally that this is due to the circle button needing to adjust size based on the font size.